src/_test_/App.test.js

```diff
@@ -0,0 +1,42 @@
+import { create } from 'react-test-renderer';
```

Comment:
- [ ] It would be better if we update the README to include instructions about how to run the test, title and description